src/test/regress/citus_tests/common.py

```diff
@@ -327,6 +327,20 @@ def stop(node_name):
             stop(node_name)
 
 
+def get_version_number(version):
+    return re.findall(r"\d+.\d+", version)[0]
+
+
+def get_actual_citus_version(pg_path, port):
+    citus_version = utils.psql_capture(pg_path, port, CITUS_VERSION_SQL)
+    citus_version = citus_version.decode("utf-8")
+    return get_version_number(citus_version)
+
+
+def versiontuple(v):
+    return tuple(map(int, (v.split("."))))
+
+
```

Comment:
Can't we get rid of those functions if we change CITUS_VERSION_SQL as
`SELECT (split_part(extversion, '.', 1)::int >= 11) as has_mx FROM pg_extension WHERE extname = 'citus';`